cmd/neofs-adm/internal/modules/morph/initialize_nns.go

```diff
@@ -13,6 +13,7 @@ import (
 	"github.com/nspcc-dev/neo-go/pkg/io"
 	"github.com/nspcc-dev/neo-go/pkg/rpcclient"
 	"github.com/nspcc-dev/neo-go/pkg/rpcclient/invoker"
+	nns2 "github.com/nspcc-dev/neo-go/pkg/rpcclient/nns"
```

Comment:
Maybe we can remove `github.com/nspcc-dev/neofs-contract/nns` import? It's used for type definitions only (and can't be used in any other way, it's a contract which shouldn't be imported at all) and we have them in the `rpcclient/nns` as well.

Reply:
I agree, I was just going to create an issue tomorrow

Reply:
```
--- a/cmd/neofs-adm/internal/modules/morph/initialize_nns.go
+++ b/cmd/neofs-adm/internal/modules/morph/initialize_nns.go
@@ -13,7 +13,7 @@ import (
        "github.com/nspcc-dev/neo-go/pkg/io"
        "github.com/nspcc-dev/neo-go/pkg/rpcclient"
        "github.com/nspcc-dev/neo-go/pkg/rpcclient/invoker"
-       nns2 "github.com/nspcc-dev/neo-go/pkg/rpcclient/nns"
+       "github.com/nspcc-dev/neo-go/pkg/rpcclient/nns"
        "github.com/nspcc-dev/neo-go/pkg/rpcclient/unwrap"
        "github.com/nspcc-dev/neo-go/pkg/smartcontract/callflag"
        "github.com/nspcc-dev/neo-go/pkg/util"
@@ -21,7 +21,6 @@ import (
        "github.com/nspcc-dev/neo-go/pkg/vm/opcode"
        "github.com/nspcc-dev/neo-go/pkg/vm/stackitem"
        "github.com/nspcc-dev/neo-go/pkg/vm/vmstate"
-       "github.com/nspcc-dev/neofs-contract/nns"
        morphClient "github.com/nspcc-dev/neofs-node/pkg/morph/client"
 )
 
@@ -281,7 +280,7 @@ func nnsIsAvailable(c Client, nnsHash util.Uint160, name string) (bool, error) {
        switch c.(type) {
        case *rpcclient.Client:
                invkr := invoker.New(c, nil)
-               reader := nns2.NewReader(invkr, nnsHash)
+               reader := nns.NewReader(invkr, nnsHash)
                return reader.IsAvailable(name)
        default:
                b, err := unwrap.Bool(invokeFunction(c, nnsHash, "isAvailable", []interface{}{name}, nil))
```
?